pcawg_metadata_parser/generate_QC_reports.py

```diff
@@ -596,6 +596,57 @@
      }
 },
 
+#query 11: get donors having fixed or version higher than 1.0.4 DKFZ/EMBL
+{
+  "name": "dkfz_embl_merged_and_higher_versions",
```

Comment:
Yeah. Sounds good. Have modified and push it to github.

-Linda







On Oct 6, 2015, at 10:42 AM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


In pcawg_metadata_parser/generate_QC_reports.py<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/38#discussion_r41272876>:

> @@ -596,6 +596,57 @@
>       }
>  },
>
> +#query 11: get donors having fixed or version higher than 1.0.4 DKFZ/EMBL
> +{
> +  "name": "dkfz_embl_merged_and_higher_versions",


The name does not quite make sense. Can we say something like this? dkfz_embl_single_gnos_uploads

—
Reply to this email directly or view it on GitHub<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/38/files#r41272876>.